app/engine/game.py

```diff
@@ -988,7 +999,7 @@ def display_payout(self) -> None:
                             for stamp in stamps.fetch_by_penguin_id(client.pid, 60)
                         ],
                         "xpStart": client.object.snow_ninja_progress,
-                        "xpEnd": exp_gained if result_rank < 24 else 100,
+                        "xpEnd": exp_percentage if result_rank < 24 else 100,
```

Comment:
Tracking update: in conversations in DM, here we should send the total current exp percentage (also if it is higher than 100)

![image](https://github.com/Lekuruu/snowflake/assets/46636310/23ef3b90-8bd1-4cb3-b9d2-6cad0025dbb7)

Reply:
Done.